### PR TITLE
Add no show flag to visualizer in exportable code

### PR DESCRIPTION
**Pull request by sovrasov**

### CODE REVIEW

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
Build # 509, see \*exportable_code\* test cases.

File: ote_sdk/ote_sdk/usecases/exportable_code/demo/demo.py [hunk 1]
```diff
@@ -64,6 +64,12 @@ def build_argparser():
         default=False,
         action="store_true",
     )
+    args.add_argument(
+        "--no_show",
```
Comment (eugene123tw) [inline ote_sdk/ote_sdk/usecases/exportable_code/demo/demo.py:68]: Hi Vlad, the code looks good to me, that said, I have a few minor change requests
* Could we change `--no_show` to `--no-show`?

Regarding README: https://github.com/openvinotoolkit/training_extensions/blob/develop/ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md
* Add descriptions for the new flag `--no-show`
* Change `<path_to_model>` to `<path_to_model_directory>`

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
@goodsong81 @eugene123tw I found the reason why openvino_deploy tests are failing: these tests use a fixed commit of [ote_sdk](https://github.com/openvinotoolkit/training_extensions/blob/develop/ote_sdk/ote_sdk/usecases/exportable_code/demo/requirements.txt). I propose to merge this PR first and then update the ote commit in requirements in a standalone PR. This allows to avoid mentioning intermediate SDK commits. As a second option I can use the latest commit from the current branch, then I don't need to create one more PR.

**Comment by eugene123tw** [comment]
> 

I think it's fine if we update OTE commit in this PR.

**Comment by sovrasov** [comment]
run ote-test

**Comment by sovrasov** [comment]
Tests are passed [#PR1204-534-test]

**Review by goodsong81 — DISMISSED** [review]
LGTM

**Comment by sovrasov** [comment]
@eugene123tw could you have a look?

File: ote_sdk/ote_sdk/usecases/exportable_code/visualizers/visualizer.py [hunk 2]
```diff
@@ -93,10 +104,14 @@ def show(self, image: np.ndarray) -> None:
...
 
     def is_quit(self) -> bool:
         """
         Check user wish to quit
         """
-        return ord("q") == cv2.waitKey(self.delay)
+        if self.no_show:
+            return False
+
+        return ord("q") == cv2.waitKey(self.delay)
```
Comment (eugene123tw) [inline ote_sdk/ote_sdk/usecases/exportable_code/visualizers/visualizer.py]: @sovrasov LGTM. Could you add a newline here?

**Review by eugene123tw — CHANGES_REQUESTED** [review]

**Review by eugene123tw — DISMISSED** [review]
LGTM

**Comment by sovrasov** [comment]
@goodsong81 could you merge?

**Review by goodsong81 — DISMISSED** [review]
LGTM. BTW, does this PR need to be included SC release?
Then please redirect to releases/v0.3.0-sc1.2 branch.

**Comment by sovrasov** [comment]
This PR could be included indeed, I'll try to redirect then, if there are no conflicts

**Comment by samet-akcay** [comment]
> LGTM. BTW, does this PR need to be included SC release?
Then please redirect to releases/v0.3.0-sc1.2 branch.

Also another PR to the [otx branch](https://github.com/openvinotoolkit/training_extensions/tree/feature/otx) please

**Comment by sovrasov** [comment]
@goodsong81 I've changed the base branch, there are no conflicts

**Review by goodsong81 — APPROVED** [review]

**Review by eugene123tw — APPROVED** [review]
LGTM

**Pull request merged**